specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2023-09-15/cosmos-db.json

```diff
@@ -7187,6 +7192,11 @@
         "enableBurstCapacity": {
           "description": "Flag to indicate enabling/disabling of Burst Capacity Preview feature on the account",
           "type": "boolean"
+        },
+        "customerManagedKeyStatus": {
+          "description": "Indicates the status of the Costumer Managed Key feature on the account. In case there are errors, the property provides troubleshooting guidance.",
+          "type": "string",
```

Comment:
Why was this marked as enum? These are a set of long strings, and modeling these as status messages in first place seems to be an incorrect outcome. Even in our code, CustomerManagedKeyStatus is modeled as string. RP code enum to specification strings have been a persistent issue and should be solved, but RP code string to specification enum should never be done.
I searched for DataEncryptionKeyStatus in our code, but couldn't see a similarity there. Will setup a meeting with Adrian and team to see how we can fix this, since it's our stable swagger.